### PR TITLE
Add snapshot tests for other subcommands `--help` output

### LINKED ISSUE #1000: Add snapshot tests for other subcommands `--help` output

### PR DESCRIPTION
**Pull request by dmitrylavrenov**
Closes #1000

### CODE REVIEW

**Review by MOZGIII — COMMENTED** [review]
We probably want to place those fixtures into a subdir

**Comment by dmitrylavrenov** [comment]
> We probably want to place those fixtures into a subdir

Reimplemented a little bit and moved to separate subdirectories all help related commands.

File: utils/e2e-tests/bash/tests/base/help.sh [hunk 1]
```diff
@@ -3,23 +3,44 @@ set -euo pipefail
...
+    -I '^humanode-peer {{sha}}$'        # template
+    -I '^humanode-peer [0-9a-f]\{40\}$' # expected
+  )
+
+  if ! DIFF="$(diff "${DIFF_CMD_ARGS[@]}" <(printf '%s' "$TEMPLATE") <(printf '%s' "$OUTPUT"))"; then
+    printf "Output did not match:\n%s\n" "$DIFF"
+    exit 1
+  fi
+
+  local SUBCOMMANDS=($(awk '/Commands:/,/Options:/{if(/Commands:|Options:/) next; print}' <(printf '%s' "$OUTPUT") | awk '{if ($1) print $1}'))
```
Comment (MOZGIII) [inline utils/e2e-tests/bash/tests/base/help.sh:33]: Why the `awk` command chaining here?

It seems to me like the `awk` invocation here would look better if we extract it into a separate fn.

Same applied to the `if`/`diff` clause.

I'd refactor this a bit further to simplify the subsequent maintenance.

Comment (MOZGIII) [reply utils/e2e-tests/bash/tests/base/help.sh:33]: Also, there are a ton of shellcheck warnings:
image

Comment (MOZGIII) [reply utils/e2e-tests/bash/tests/base/help.sh]: Refactored a lot of stuff, but still left the chained `awk` invocations as is for you to look into.

Comment (dmitrylavrenov) [reply utils/e2e-tests/bash/tests/base/help.sh]: Thanks for the improvements!

The `awk` usage has been refactored - https://github.com/humanode-network/humanode/pull/1034/commits/66cfa4912f927af83ab4796e80906785159978b6.

**Review by MOZGIII — APPROVED** [review]

**Pull request merged**